src/Umbraco.Web.UI.Client/bower.json

```diff
@@ -21,7 +21,7 @@
         "underscore": "~1.7.0",
         "rgrove-lazyload": "*",
         "bootstrap-social": "~4.8.0",
-        "jquery": "2.0.3",
+        "jquery": "2.1.4",
```

Comment:
Are we keeping this version or not? @perploug  said yesterday it would be reverted? If not, an issue needs to be logged in the tracker